### PR TITLE
feat: welcome message

### PR DESCRIPTION
**Pull request by StrangeGirlMurph**
:)

### CODE REVIEW

**Comment by StrangeGirlMurph** [comment]
yeah I know... the merge history is ugly... I really need to learn how to actually merge/rebase....

File: src/events/guildMemberAdd.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import { EmbedBuilder, GuildMember, inlineCode } from "discord.js";
+import { addDefaultEmbedFooter } from "../util/embeds";
+import { logger } from "../logger";
+import { config } from "../config";
+
+export default async function guildMemberAdd(member: GuildMember) {
+	let embed = new EmbedBuilder()
+		.setTitle(`Hey ${member.displayName} 😊`)
+		.setThumbnail(member.guild.iconURL({ size: 1024 }) || "")
+		.setDescription(`You are the ${member.guild.memberCount}th member of this server!`)
```
Comment (Chicken) [inline src/events/guildMemberAdd.ts:10]: The suffix is not always "th"

File: src/events/guildMemberAdd.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+		.addFields(
+			{
+				name: `Welcome on ${member.guild.name}!`,
```
Comment (Chicken) [inline src/events/guildMemberAdd.ts:13]: Welcome to*

File: src/events/guildMemberAdd.ts [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+			{
+				name: `I am ${config.botName} <3`,
+				value: `I manage this whole server from creating polls to welcoming new members like you. I have a lot of cool features. You can find out more about them by using ${inlineCode(
+					"/help"
+				)} in any channel. If I need something from you I'll ping you like this <@${member.user.id}> :)`,
+			}
+		);
+	embed = addDefaultEmbedFooter(embed);
+
+	logger.debug("Someone new joined the guild");
```
Comment (Chicken) [inline src/events/guildMemberAdd.ts:25]: Just remove the debug or at least make it useful by including some ids or anything

**Review by Chicken — CHANGES_REQUESTED** [review]

File: src/events/guildMemberAdd.ts (hunk 1 above)
Comment (StrangeGirlMurph) [reply src/events/guildMemberAdd.ts:10]: ^^ true just go ahead and fix it

File: src/events/guildMemberAdd.ts (hunk 3 above)
Comment (StrangeGirlMurph) [reply src/events/guildMemberAdd.ts:25]:
```suggestion
```

File: src/events/guildMemberAdd.ts (hunk 2 above)
Comment (StrangeGirlMurph) [reply src/events/guildMemberAdd.ts:13]:
```suggestion
				name: `Welcome to ${member.guild.name}!`,
```

**Review by Chicken — APPROVED** [review]

**Review by IanToujou — APPROVED** [review]
Nya nya nya

**Pull request merged**